testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_cbl_log_redaction.py

```diff
@@ -120,45 +124,59 @@ def test_verify_invalid_maskPassword_in_logs(params_from_base_test_setup, invali
     sg_client.create_user(sg_admin_url, sg_db, "autotest", password="password", channels=channels)
     sg_client.create_session(sg_admin_url, sg_db, "autotest")
 
-    replicator_authenticator = authenticator.authentication(username="autotest", password=invalid_password, authentication_type="basic")
+    replicator_authenticator = authenticator.authentication(username="autotest", password=invalid_password,
+                                                            authentication_type="basic")
     repl_config = replicator.configure(cbl_db, target_url=sg_blip_url, continuous=False,
                                        channels=channels, replicator_authenticator=replicator_authenticator)
 
     repl = replicator.create(repl_config)
     replicator.start(repl)
     replicator.wait_until_replicator_idle(repl, err_check=False)
-
-    verify_password_masked(liteserv_platform, log_file, invalid_password)
+    verify_password_masked(liteserv_platform, log_file, invalid_password, testserver, log_filename)
 
 
-def verify_password_masked(liteserv_platform, log_file, password):
-    if liteserv_platform.lower() == "android":
-        log_file = log_file.replace(" ", "\ ")
-
-    if liteserv_platform == "net-msft":
-        log_full_path = "/tmp/cbl-logs/"
+def verify_password_masked(liteserv_platform, log_file, password, testserver, log_filename):
+    """
+    @note: Porting logs for Android, xamarin-android, net-core and net-uwp platform, as the logs reside
+           outside runner's file directory
+    """
+    log_full_path = "/tmp/cbl-logs/"
+    os.mkdir(log_full_path)
+    if liteserv_platform.lower() == "android" or liteserv_platform.lower() == "xamarin-android":
+        log_info("Running Command: 'adb pull {} {}".format(log_file, log_full_path))
+        cmd = ["adb", "pull", log_file, log_full_path]
```

Comment:
as we run sometimes on device and sometimes on emulator, this will fail if just adb command runs. you can '-d' if it is device enabled otherwise '-e'

Reply:
done